Source/EasyNetQ.Tests/project.json

```diff
@@ -0,0 +1,39 @@
+{
+  "version": "2.0.2-netcore0001",
```

Comment:
same comment about updating the description, authors, tags to be like the one they have here: https://github.com/EasyNetQ/EasyNetQ/blob/feature/netcore/Source/EasyNetQ/project.json